src/lib/locationHelpers.ts

```diff
@@ -32,6 +38,7 @@ export const getLocationArgs = async (
 
   return {
     near: isString(near) ? near : `${location.lat},${location.lng}`,
-    max_distance: location.maxDistance || DEFAULT_MAX_DISTANCE_KM,
+    max_distance:
+      maxDistance || location.maxDistance || DEFAULT_MAX_DISTANCE_KM,
```

Comment:
Because `near` is just a simple string in some cases, we need to be able to pass `maxDistance` separately.